packages/libsql-client/src/sqlite3.ts

```diff
@@ -185,9 +186,13 @@ export class Sqlite3Client implements Client {
         }
     }
 
-    async sync(): Promise<void> {
+    async sync(): Promise<Replicated> {
         this.#checkNotClosed();
-        await this.#getDb().sync();
+        const rep = (await this.#getDb().sync()) as any;
```

Comment:
had to convert to any because the types in libsql-js 0.4 didn't work, will fix that in 0.4.1 but this works anyways for now.

Reply:
solved in https://github.com/tursodatabase/libsql-client-ts/pull/246/commits/73ec2956b2de1393e8339744369c1fae7db11c1e